modules/reindex/src/test/resources/rest-api-spec/test/reindex/95_remote_auth.yaml

```diff
@@ -0,0 +1,101 @@
+---
+"Reindex from remote understands unauthorized":
```

Comment:
These all need to skip if the client doesn't support embedded stash keys and I'll add that when I next switch to the branch.